src/frontend/src/lib/components/dapps/DappCard.svelte

```diff
@@ -30,7 +30,9 @@
 				{oneLiner}
 			</p>
 		</section>
-		<section class="absolute bottom-4 left-4 right-4 max-h-6 min-h-6 overflow-hidden md:max-h-14">
+		<section
+			class="absolute bottom-4 left-4 right-4 max-h-6 min-h-6 overflow-hidden md:max-h-[calc(3.5rem+1px)]"
```

Comment:
> there is a slight issue with not visible borders

Instead of adding 1px to the height, should we add a border-1 transparent given the description of the issue?

Reply:
Hmm, I don't get this to work with the overflow hidden. Do you want to give it a try?

Reply:
There you go: https://github.com/dfinity/oisy-wallet/pull/3446